core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -871,7 +874,7 @@ public PropServerConfiguration(
             this.o3PurgeDiscoveryQueueCapacity = Numbers.ceilPow2(getInt(properties, env, PropertyKey.CAIRO_O3_PURGE_DISCOVERY_QUEUE_CAPACITY, 128));
             this.o3ColumnMemorySize = (int) Files.ceilPageSize(getIntSize(properties, env, PropertyKey.CAIRO_O3_COLUMN_MEMORY_SIZE, 8 * Numbers.SIZE_1MB));
             this.maxUncommittedRows = getInt(properties, env, PropertyKey.CAIRO_MAX_UNCOMMITTED_ROWS, 500_000);
-            this.commitLag = getLong(properties, env, PropertyKey.CAIRO_COMMIT_LAG, 300_000) * 1_000;
+            this.o3MaxLag = getLong(properties, env, PropertyKey.CAIRO_COMMIT_LAG, 10 * Dates.MINUTE_MILLIS) * 1_000;
```

Comment:
Key should be renamed.

Reply:
yeah, i thought that too, but we have to maintain backward compatibility

Reply:
renamed all `commitLag` occurrences in https://github.com/questdb/questdb/pull/2759/commits/be85cbfb3d59b70be8180622869ea92a58033841